test/expect.js

```diff
@@ -397,6 +397,35 @@ describe('expect', function () {
     err(function(){
       expect({foo: 'bar'}).to.be.empty;
     }, "expected { foo: \'bar\' } to be empty");
+
+    err(function(){
+      expect(0).to.be.empty;
+    }, "expected 0 to be empty");
```

Comment:
I wasn't sure about this when I added it, and kind of put it here for discussion. It was more because the empty seemed like a means to check something has no properties or length. Happy to remove that though if its deemed ok for numbers to pass.